presto-hive/src/main/java/io/prestosql/plugin/hive/HiveWriterFactory.java

```diff
@@ -589,7 +588,7 @@ private void validateSchema(Optional<String> partitionName, Properties schema)
                         schemaName,
                         tableName,
                         inputHiveType,
-                        partitionName,
+                        partitionName.orElse(""),
```

Comment:
i am not sure i like `.. but partition '' declared column` in the final err msg.

but, the `partitionName` must exist at this point, so `partitionName.orElseThrow` would be appropriate

Reply:
I think that’s true, but I’m not comfortable with restructuring the code so the optional can be removed (especially without tests).

Reply:
I think that’s true, but I’m not comfortable with restructuring the code so the optional can be removed (especially without tests).

Reply:
Can you add a code comment maybe like `// partitionName is expected to exist`` ?